script/constants/buckets.js

```diff
@@ -0,0 +1,21 @@
+const {
+  DEVELOPMENT,
+  PREVIEW,
+  PRODUCTION,
+  STAGING,
+  VAGOVDEV,
+  VAGOVSTAGING,
+} = require('./environments');
+
+const hostnames = require('./hostnames');
+
+const bucket = 'https://s3-us-gov-west-1.amazonaws.com';
+
+module.exports = {
+  [DEVELOPMENT]: `${bucket}/${hostnames[DEVELOPMENT]}`,
+  [PREVIEW]: `${bucket}/${hostnames[PREVIEW]}`,
+  [PRODUCTION]: `${bucket}/${hostnames[PRODUCTION]}`,
+  [STAGING]: `${bucket}/${hostnames[STAGING]}`,
+  [VAGOVDEV]: `${bucket}/${hostnames[VAGOVDEV]}`,
+  [VAGOVSTAGING]: `${bucket}/${hostnames[VAGOVSTAGING]}`,
+};
```

Comment:
This was overkill, but I could see this being useful for the CMS Content Deployment.

Reply:
@todo Bucket names for VA environments are changing because `.` can't be in the name